modules/test/playwright/pages/accessibility-menu-web/AccessibilityMenuPage.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export class AccessibilityMenuPage {
+	readonly closeButton: Locator;
+	readonly enableCheckbox: Locator;
+	readonly menuButton: Locator;
+	readonly page: Page;
+	readonly switchButton: Locator;
+	readonly updateButton: Locator;
+
+	constructor(page: Page) {
+		this.closeButton = page.getByLabel('close', {exact: true});
+		this.enableCheckbox = page.getByLabel('Enable Accessibility Menu');
+		this.menuButton = page.getByRole('button', {
+			name: 'Open Accessibility Menu',
+		});
+		this.page = page;
+		this.switchButton = page.getByLabel('Underlined Links');
```

Comment:
We have multiple switch buttons in the accessibility menu, so we need to be more specific. Also we usually call this a `toggle` instead of a switch. So `underlinedLinksToggle` would be better.